src/libstd/sys/windows/fs.rs

```diff
@@ -575,3 +575,12 @@ pub fn canonicalize(p: &Path) -> io::Result<PathBuf> {
         PathBuf::from(OsString::from_wide(buf))
     })
 }
+
+pub fn copy(from: &Path, to: &Path) -> io::Result<u64> {
+    let pfrom = to_utf16(from);
+    let pto = to_utf16(to);
+    try!(cvt(unsafe {
+        c::CopyFileW(pfrom.as_ptr(), pto.as_ptr(), libc::FALSE)
+    }));
+    stat(to).map(|attr| attr.size())
```

Comment:
Hm well no nevermind, there's a few steps taken in the unix implementation (e.g. copy then set permissions), so this may not be so bad after all.